docs/MSPX.md

```diff
@@ -24,28 +26,87 @@ MspX is enabled by this setting in the receiver:
 
 For enabling MSP-RC set:
 
-- "Rx Snd RcChannel" = "rc override" or "rc channels"
+- "Rx Snd RcChannel" = "rc override" or "rc channels" (both have the same functionality in mspX mode)
 
 With MSP-RC enabled, SET_RAW_RC messages are sent to the flight controller. In the flight controller configure the receiver to the MSP protocol. This allows one to avoid the extra wire for CRSF or SBus. MspX needs to be enabled.
 
+***Notes***:
+RC Link Statistics are only send to the Flight controller via MSP, if "rc override" or "rc channels" is enabled. 
+MSP-RC messages will always override other RC Link inputs. If CRSF is connected and CRSF selected as the RC Protocol in INAV, MSP-RC still has priority if enabled in mLRS and connected to a MSP Enabled UART.
+
 ## INAV Configuration
 
-The baudrate should be set to 115200 as lower baudrates have shown irregular telemetry update rates on the radio due to the lack of flow control on the INAV side. 
+The baudrate should be set to 115200 or higher for consistent dataflow and RC control. Baudrates of less than 57600 can cause message loss on the INAV side and result in a inconsistent RC control at 50Hz RC Rate, if MSP-RC is used (INAV Limitation). 
 
-INAV 8.0 and later will also support a baudrate of 230400 but no change in performance or stability was noticed. Additionally with INAV 8.0 the performance of the serial link to a ground control station will be increased, due to some optimizations in INAV. 
+INAV 8 and later will also support a baudrate of 230400 but no change in performance or stability was noticed. Additionally with INAV 8 the performance of the serial link to a ground control station will be increased, due to some optimizations in INAV. 
 
 To use a mLRS receiver with INAV in MspX mode, the following settings have to be applied in INAV:
 - Enable MSP for the Serial Port the mLRS receiver is connected to (UART 2 is recommended on most STM32 flight controllers; do not enable "Serial RX").
 - Set the baudrate to 115200 or higher.
   
 <img src="images/MSPX_ports.png" width="720px">
 
-- In the Receiver tab, select the Receiver Type in the Receiver Mode panel to MSP, and save settings.
+- In the Receiver tab, select the Receiver Type in the Receiver Mode panel to MSP and also set the RSSI Source to MSP (INAV 8 and later) and save settings.
```

Comment:
Nitpick, grammar:

'In the Receiver tab, change the Receiver Mode to 'MSP' and the RSSI Source to 'MSP' then save settings.'